.github/workflows/ci.yml

```
@@ -2,18 +2,48 @@ name: Build and Release
 on: push
 
 jobs:
-  weasyprint-build:
-    name: Build WeasyPrint
-    runs-on: ubuntu-latest
+  weasyprint-build-arm64:
+    name: Build WeasyPrint arm64
+    runs-on: ubuntu-24.04-arm
+    strategy:
+      matrix:
+        version: ["3.12", "3.13"]
```

can we use matrix here?
```suggestion
    strategy:
      matrix:
        version: ["3.12", "3.13"]
        arch: ["x86_64", "arm64"]
    runs-on: ${{ matrix.arch == 'x86_64' && 'ubuntu-24.04' || 'ubuntu-24.04-arm' }}
```